chain/chain_config.go

```diff
@@ -78,6 +80,7 @@ type Config struct {
 
 	//zkEVM updates
 	MordorBlock *big.Int `json:"mordorBlock,omitempty"`
+	Fork7Block  *big.Int `json:"fork7Block,omitempty"`
```

Comment:
I think its better to still call it "forkEtrog" because zkEVM has a story of shifting forkID numbers (etrog initially was supposed to be forkID5).

Reply:
Is it better eg. `Fork7EtrogBlock` and `Fork5MordorBlock`?  Mordor and Etrog is a little bit unclear without fork id.
Here is the version list: https://github.com/0xPolygonHermez/cdk-erigon/issues/37#issuecomment-1891244596